models/marts/sales/sales.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+models:
+    - name: sales__by_customer_and_item
+      description: "A sales data model, to be queried/aggregated further by BI tools. The grain is each individual item sold."
```

Comment:
It looks like you did not keep the unique id in order_items, I personally did because I think each table or view should have a clear unique id and tests should look for that.